releasenotes/notes/gradient_wrapper_param_binding-44a4aaf85e9a85c2.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Can you remove the releasenote? We'll do one for all of the stuff that moved from Aqua to Terra 🙂

Reply:
Sure.